teleop_twist_keyboard.py

```diff
@@ -19,18 +19,30 @@
 e/c : increase/decrease only angular speed by 10%
 anything else : stop
 
+For Horonomic base, holding down Shift key
+
+For Drones, use t/b to control elevation
```

Comment:
t is up, increase z value and b for another, how about
`for Drones, use t for up and b for down`
??

--
◉ Kei Okada


On Sun, Jan 24, 2016 at 5:43 AM, Austin <notifications@github.com> wrote:

> In teleop_twist_keyboard.py
> <https://github.com/ros-teleop/teleop_twist_keyboard/pull/5#discussion_r50625274>
> :
>
> > @@ -19,18 +19,30 @@
> >  e/c : increase/decrease only angular speed by 10%
> >  anything else : stop
> >
> > +For Horonomic base, holding down Shift key
> > +
> > +For Drones, use t/b to control elevation
>
> These instructions are pretty vague; they still leave open questions about
> which keys are active when holding shift, and how t and b control
> elevation.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ros-teleop/teleop_twist_keyboard/pull/5/files#r50625274>
> .
>